Makefile

```diff
@@ -98,11 +98,10 @@ operator-container: build
 	fi
 	$(IMAGE_BUILD_CMD) build --no-cache -f openshift-ci/Dockerfile.deploy -t $(FULL_OPERATOR_IMAGE) --build-arg BIN_DIR="_output/bin/" --build-arg ASSETS_DIR="assets" build/
 
-.PHONY: metadata-container
-metadata-container: generate-latest-dev-csv
+.PHONY: bundle-container
+bundle-container: generate-latest-dev-csv
 	@echo "Building the performance-addon-operator metadata image"
-	@find build/_output/olm-catalog/ -type f -exec sed -i "s|REPLACE_IMAGE|${FULL_OPERATOR_IMAGE}|g" {} \; || :
-	$(IMAGE_BUILD_CMD) build --no-cache -f openshift-ci/Dockerfile.metadata -t "$(FULL_METADATA_IMAGE)" .
+	$(IMAGE_BUILD_CMD) build --no-cache -f openshift-ci/Dockerfile.bundle -t "$(FULL_METADATA_IMAGE)" .
```

Comment:
lets change *METADATA_IMAGE  to *BUNDLE_IMAGE for consistency

Reply:
done!